higher_health/tests/test_views.py

```diff
@@ -0,0 +1,191 @@
+
+        self.assertEqual(response.status_code, 200)
+
+        errors = response.context["form"].errors
+        self.assertEqual(errors["msisdn"], ["This field is required."])
+        self.assertEqual(errors["first_name"], ["This field is required."])
+        self.assertEqual(errors["last_name"], ["This field is required."])
+        self.assertEqual(errors["age_range"], ["This field is required."])
+        self.assertEqual(errors["gender"], ["This field is required."])
+        self.assertEqual(errors["province"], ["This field is required."])
+        self.assertEqual(errors["latitude"], ["This field is required."])
+        self.assertEqual(errors["longitude"], ["This field is required."])
+        self.assertEqual(errors["address"], ["This field is required."])
+        self.assertEqual(errors["symptoms_fever"], ["This field is required."])
+        self.assertEqual(errors["symptoms_cough"], ["This field is required."])
+        self.assertEqual(errors["symptoms_sore_throat"], ["This field is required."])
+        self.assertEqual(
+            errors["symptoms_difficulty_breathing"], ["This field is required."]
+        )
+        self.assertEqual(errors["symptoms_muscles_hurt"], ["This field is required."])
+        self.assertEqual(errors["symptoms_taste"], ["This field is required."])
+        self.assertEqual(errors["medical_exposure"], ["This field is required."])
+        self.assertEqual(
+            errors["medical_pre_existing_condition"], ["This field is required."]
+        )
+        self.assertEqual(
+            errors["medical_confirm_accuracy"], ["This field is required."]
+        )
+
+    def test_post_with_invalid_accuracy(self):
+        data = get_data()
+        data["medical_confirm_accuracy"] = "no"
+        response = self.client.post(reverse("healthcheck_questionnaire"), data)
+
+        self.assertEqual(response.status_code, 200)
+
+        errors = response.context["form"].errors
+        self.assertEqual(
+            errors["medical_confirm_accuracy"],
+            ["You need to confirm that this information is accurate"],
+        )
+
+    @responses.activate
+    def test_post_get_coordinates_from_address(self):
+        data = get_data()
+        data["latitude"] = ""
+        data["longitude"] = ""
+        data["address"] = "4 friend street woodstock"
+
+        responses.add(
+            responses.GET,
+            "https://maps.googleapis.com/maps/api/place/findplacefromtext/json?key=TEST_API_KEY&input=4+friend+street+woodstock&inputtype=textquery&language=en&fields=geometry",
+            json={
+                "candidates": [
+                    {"geometry": {"location": {"lat": "11.1", "lng": "22.2"}}}
+                ]
+            },
+            status=200,
+            match_querystring=True,
+        )
+
+        response = self.client.post(reverse("healthcheck_questionnaire"), data)
+
+        self.assertEqual(response.status_code, 302)
```

Comment:
Should we maybe also assert that the `responses` mock was called, and that the database object was created with the correct values?